hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenUpdateHandler.java

```diff
@@ -363,7 +365,7 @@ private void updateExpiryFields(
             builder.autoRenewSecs(resolvedExpiry.autoRenewPeriod());
         }
         if (op.hasAutoRenewAccount()) {
-            builder.autoRenewAccountNumber(resolvedExpiry.autoRenewNum());
+            builder.autoRenewAccountId(AccountID.newBuilder().accountNum(resolvedExpiry.autoRenewNum()));
```

Comment:
Need to modify `ExpiryMeta` to have `autoRenewAccountId` . So we can directly use it.

Reply:
This will be done in #7350.